app/domain/operations/transmittable/generate_request_objects.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+module Operations
+  module Transmittable
+    # find or create job, then create response transmission/transaction, save payload and link to subject
+    class GenerateRequestObjects
+      include ::Operations::Transmittable::TransmittableUtils
+
+      attr_reader :transaction, :transmission
+
+      def call(params)
+        values = yield validate(params)
+        job_params = yield construct_job_params
+        @job = yield create_job(job_params)
+        transmission_params = yield construct_transmission_params(values)
+        @transmission = yield create_request_transmission(transmission_params, @job)
+        @family = yield create_family(values)
+        transaction_params = yield construct_transaction_params
+        @transaction = yield create_request_transaction(transaction_params, @job)
```

Comment:
@jacobkagon We discussed having multiple transactions, one for the Account and one per contact. I do not see that here.